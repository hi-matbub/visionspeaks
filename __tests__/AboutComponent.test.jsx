import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import About from '../lib/static/About.mdx';

it('AboutComponent component renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<About />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('AboutComponent component renders unchanged', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});