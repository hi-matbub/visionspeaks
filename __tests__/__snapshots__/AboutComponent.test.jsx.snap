// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutComponent component renders unchanged 1`] = `
<h2>
  About us
</h2>
`;
